packages/wallets/src/features/cashier/modules/Transactions/Transactions.scss

```diff
@@ -8,26 +8,22 @@
 
     // ignore CashierContent top padding
     margin-top: -4.8rem;
+
     @include mobile {
         margin-top: -2.4rem;
     }
 
-    // adjust filter dropdown width
-    .wallets-dropdown {
+    &__dropdown {
         max-width: 19.5rem;
+
         @include mobile {
             max-width: unset;
         }
-    }
-
-    // don't need details/error message
-    .wallets-textfield__message-container {
-        display: none;
-    }
 
-    // don't show text selection
-    .wallets-textfield__field::selection {
-        background: none;
+        /* stylelint-disable-next-line selector-class-pattern */
+        .deriv-dropdown__items--xs {
+            max-height: unset;
+        }
```

Comment:
Resolve issue for dropdown where full list is not seen for transactions page:
<img width="512" alt="image" src="https://github.com/user-attachments/assets/5efce849-2a2a-43b8-8ab5-1c2819212b20">